### DISCUSSION TITLE
Example .yml for mullvald

### DISCUSSION THREAD

**Discussion by Rurushu9** [haugene/docker-transmission-openvpn#2682] [Q&A]
Can someone post an example docker-compose.yml for transmission-vpn with mullvald

**Comment by ilike2burnthing** [comment]
https://github.com/haugene/docker-transmission-openvpn/blob/master/docker-compose.yml
plus
https://haugene.github.io/docker-transmission-openvpn/provider-specific/#mullvad_ovpn
